keras_core/trainers/trainer.py

```diff
@@ -18,6 +18,7 @@ def __init__(self):
         self._run_eagerly = False
         self._jit_compile = None
         self.compiled = False
+        self.loss = {}
```

Comment:
Should be `None` no?

Reply:
In `tf.keras`, it is `{}`.
https://colab.research.google.com/drive/1DEr6w72wwuNGR9MqNlB_sgTenSBdLBge?resourcekey=0-4Yj0umES4aNjX2s0Qc6gTA&usp=sharing

Would you like it to be `None`?
I am OK with either one.

Reply:
`None` is the signature default so it's much more sensible than an empty dicts (`loss` can be a dict but it's not a common case).

Reply:
Changed to `None`.